word2vec.ipynb

```diff
@@ -0,0 +1,1140 @@
+{
```

Comment:
<p>You can remove these duplicate imports (they're already in the cell above)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CallmeMehdi/swahili-word-embedding/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>